platform/ui/src/views/StudyList/components/StudyListFilter.js

```diff
@@ -1,7 +1,103 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const StudyListFilter = () => {
-  return <div>{`<- STUDYLIST FILTER CONTENT ->`}</div>;
+import { Button, Icon, Input, Typography } from '@ohif/ui';
+
+const StudyListFilter = ({
+  patientName,
+  mrn,
+  studyDate,
+  description,
+  modality,
+  accession,
+  instances,
+  numOfStudies = 0,
+  sortByFilter,
+  onSortByChange,
+  onFilterChange,
```

Comment:
Most things like this split props:

- "Meta"
- "Value"

Where meta drives how things are displayed/formatted/rendered, and value is the value we're actively modifying and communicating to the parent.

Meta might look like:

```js
filtersMeta: [
  {
    name: 'patientName',
    displayName: 'Patient Name',
    inputType: 'text',
    isSortable: true
  }
]
```

With values similar to:

```js
filterValues: {
    // note: value is using "name" to match field meta
    patientName: "",
    mrn: "",
    studyDate: "",
    // These fields might be in a separate object
    // Probably still have to be props so we can support persisting them between routes (cached filter on back navigation)
    sort: "" // empty or field name
    sortDirection: 0, // -1, 0, 1
    page: 0,
    resultsPerPage: 25,
}
```

Reply:
Change the code to something similar to this.